apps/arena/lib/arena/game/bounties.ex

```diff
@@ -0,0 +1,78 @@
+defmodule Arena.Game.Bounties do
+  @moduledoc """
+  Module to handle quest logic in arena application
+  TODO this module shouldn't exist, this a workaround until we have a shared application to
```

Comment:
Why not add it as a dependency library?

```elixir
{:game_backend, in_umbrella: true, runtime: false}
```

Reply:
We talked about this with @Nico-Sanchez , in the end, we'll go for the option to create a new application to contain the champion's logic and both GameBackend and Arena will take the bounties logic from there.
But not in this pr, we wan't to have it in the short term